jar-infer/test-java-lib-jarinfer/src/main/java/com/uber/nullaway/jarinfer/toys/Toys.java

```diff
@@ -1,4 +1,13 @@
-package com.uber.nullaway.jarinfer.toys.unannotated;
+package com.uber.nullaway.jarinfer.toys;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
+@interface ExpectNullable {}
```

Comment:
These should be in separate files

Reply:
Keeping this as is, per our offline discussion.